### DISCUSSION TITLE
How should I get Arctic Code Vault Contributor badge ???

### DISCUSSION THREAD

**Discussion by dev-aniketj** [community/community#45154] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

How should I get Arctic Code Vault Contributor badge ???
![image](https://user-images.githubusercontent.com/81229551/214109446-1f33fbfc-2bdd-4451-86d4-1a2ed0f403ce.png)

**Comment by antonblr** [comment, marked as answer]
tl; dr; - you cannot get it anymore (unless there will be more events like that in the future).

@dev-aniketj - it has been awarded to every developer who has contributed to open source projects at the time of the snapshot (02/02/2020). You can find more info here - https://archiveprogram.github.com/arctic-vault/
Your best bet is to start contributing to open source projects and hope the event repeats.

**Reply by Janxinka** [reply]
Hm

**Comment by SilasRodrigues19** [comment]
Hey @dev-aniketj 

It was a badge for anyone who contributed to a repository in 2020 at the [GitHub Archive Program](https://archiveprogram.github.com/) event.

**Comment by dev-aniketj** [comment]
Is their is any options....???

**Reply by SilasRodrigues19** [reply]
There is no way to get these old badges, at least not yet.

**Comment by mwyau** [comment]
I used the email address in my commit without the number prefix: mwyau@users.noreply.github.com instead of 5298134+mwyau@users.noreply.github.com, in this PR: https://github.com/opengapps/opengapps/pull/639
I used to have the 2020 Arctic Code Vault badge, and it was gone at some point.
